native/acter/src/testing.rs

```diff
@@ -112,7 +112,7 @@ pub async fn ensure_user(
     };
 
     if let Some(token) = reg_token {
-        info!("Login for {username} failed: {e}. Trying to register with token instead.");
+        error!("Login for {username} failed: {e}. Trying to register with token instead.");
```

Comment:
this isn't an error-level but supported behaviour.
```suggestion
        info!("Login for {username} failed: {e}. Trying to register with token instead.");
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/b7b8a9b1a3f7b414661e413d687da13809d4bd9b